contract/contracts/PixelDevsUkraineDonation.sol

```diff
@@ -22,6 +22,7 @@ contract PixelDevsUkraineDonation is
     string public baseURI = "ipfs://abcd.../";
     uint256 public minimumMintPrice = 12 ether;
     bool public contractState = true;
+    address public withdrawWallet; 
```

Comment:
Should we add Ukraine DAO address here as default?

Reply:
Agreed, I'll hardcode the default address.